examples/topics/network_packets.ipynb

```diff
@@ -106,7 +107,10 @@
     "\n",
     "hv.extension('bokeh')\n",
     "\n",
-    "%opts RGB Graph Nodes [bgcolor='black' width=800 height=800 xaxis=None yaxis=None]"
+    "opts.defaults(\n",
+    "    opts.Graph(bgcolor='black', width=800, height=800, xaxis=None, yaxis=None),\n",
+    "    opts.Nodes(bgcolor='black', width=800, height=800, xaxis=None, yaxis=None),\n",
+    "    opts.RGB(  bgcolor='black', width=800, height=800, xaxis=None, yaxis=None))"
```

Comment:
I'd vote for:

```
o = dict(bgcolor='black', width=800, height=800, xaxis=None, yaxis=None)
opts.defaults(opts.Graph(**o), opts.Nodes(**o), opts.RGB(**o))
```

Reply:
+1, we've had some disagreement about this but in holoviews I've generally favored an approach like this.

Reply:
I really prefer to avoid `**` when possible. I would prefer:

```python
o = opts.Options(bgcolor='black', width=800, height=800, xaxis=None, yaxis=None)
opts.defaults(opts.Graph(o), opts.Nodes(o), opts.RGB(o))
```

But this isn't currently supported (but it could be). As it is, I'll implement your suggestion even though I don't hugely like it (but the current redundant version isn't great either)

Reply:
Fixed in 71f5ee6

Reply:
Sure, consider adding support for your proposed syntax to HoloViews, which could at least check if those options are valid for *some* backend. But meanwhile your fix is fine, thanks.